great-idea/css/index.css

```diff
@@ -63,4 +64,130 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+h1 {
+	display: inline-block;
+	font-size: 80px;
+	text-align: center;
+	vertical-align: top;
+	margin: 80px 50px;
+}
+
+
+
+button {
```

Comment:
Best practice to always add `cursor: pointer` on your buttons. It's good for user experience and also accessibility.